app/main.py

```diff
@@ -1 +1,48 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.appetite and self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            points = self.appetite
+            self.appetite = 0
+            return points
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True,
+                 appetite: int = 3) -> None:
```

Comment:
```suggestion
    def __init__(
        self, name: str, appetite: int = 3, is_hungry: bool = True
    ) -> None:
```
No need to change order of params